app/src/main/java/org/simple/clinic/registration/name/RegistrationFullNameScreenController.kt

```diff
@@ -45,7 +45,7 @@ class RegistrationFullNameScreenController @Inject constructor(
     return events
         .ofType<RegistrationFullNameScreenCreated>()
         .flatMap {
-          facilityRepository.facilities()
+          facilityRepository.facilitiesInAscending()
```

Comment:
Since the only thing we are using this for is to check if the facilities have been synced at least once, we should change it to just get the facility count instead of loading all facilities.

Reply:
It's a small data-set (20 rows) so the performance benefit will be negligible. I'm not very sure if I want to touch this right now.

Reply:
Agree with Vinay, but +1 to Saket. Seems like a micro-optimization to do right now.